catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -588,6 +598,39 @@ public void render() {
 			testPixels = ScreenUtils.getFrameBufferPixels(testX, testY, testWidth, testHeight, false);
 			makeTestPixels = false;
 		}
+
+		if (spriteToFocusOn != null) {
+			calculateCameraPositionForFocusedSprite();
+		}
+	}
+
+	@VisibleForTesting
+	public void calculateCameraPositionForFocusedSprite() {
```

Comment:
Please rename to:
```suggestion
	public void updateCameraPositionForFocusedSprite() {
```
as the word calculate does not imply any action only a calculation.